packages/twenty-server/src/engine/core-modules/record-crud/zod-schemas/record-input.zod-schema.ts

```diff
@@ -5,19 +5,29 @@ import { type ObjectMetadataEntity } from 'src/engine/metadata-modules/object-me
 
 export const generateRecordInputSchema = (
   objectMetadata: ObjectMetadataEntity,
+  includeId = false,
```

Comment:
If we go with proper tool definition we'll probably need to split this into two functions for update/create (but which will share common logic). The includeId felt a bit hacky already